application/F3DStarter.cxx

```diff
@@ -179,6 +179,19 @@ int F3DStarter::Start(int argc, char** argv)
       .setIcon(F3DIcon, sizeof(F3DIcon));
   }
 
+  try
+  {
+    this->Internals->Engine->loadPlugin("libf3d-plugin-core.so");
```

Comment:
@mwestphal right now the enabled plugins are hardcoded. Not sure how F3D should handle them automatically. Here are some suggestions, feel free to add more suggestion based on your experience with PV:
- Add the list of plugins in the config file
- Add an option `--load-plugin=/path/to/plugin.so`
  - The core plugin is always loaded
  - We can support `--load-plugin=assimp` and it will look for `libf3d-plugin-assimp.so`
- Search for all `libf3d-plugin-xxxxx.so` libraries and load them automatically

Reply:
- Add the list of plugins in the config file

No but yes, through the options

- Add an option --load-plugin=/path/to/plugin.so

Definitely needed, but should allow for multiple plugins:

 `--load-plugins=/path/to/plugin1.so,/path/to/plugin2.so,`
 
-  We can support --load-plugin=assimp and it will look for libf3d-plugin-assimp.so

Not against it but what would be the location in the user space ? If from install, then ok.

- Search for all libf3d-plugin-xxxxx.so libraries and load them automatically

Same question as above

- Another suggestion: supporting loading a dedicated json file containing a list of plugin with their path (think paraview.plugins file)